valhalla/baldr/json.h

```diff
@@ -47,10 +48,10 @@ class Jmap : public std::unordered_map<std::string, Value> {
 };
 
 // the array value type in json
-class Jarray : public std::list<Value> {
+class Jarray : public std::vector<Value> {
 public:
   // just specialize vector
-  using std::list<Value>::list;
+  using std::vector<Value>::vector;
```

Comment:
allows for reservation in the serializer. although we shoudl probably just convert the rest of the serializers to rapidjson